file/errorLogging.go

```diff
@@ -0,0 +1,31 @@
+package file
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+)
+
+// LogErrAsWarnExceptAsDebugIfClosingError logs an error
```

Comment:
I would keep this in elrond-go only. Is too specific

Reply:
done